src/ctelnet.cpp

```diff
@@ -283,6 +288,8 @@ void cTelnet::connectIt(const QString& address, int port)
         return;
     }
 
+    emit signal_connecting(mpHost);
```

Comment:
Ah, these signals are not required any more - I was generating Mud Server connecting/connected/disconnected signals to use the RP "elapsed" timer to show the profile connection time.  Given that there are Lua / GMCP commands that control the elapsed/remaining part of the RP this can be done in other ways as the user/package writer might want and these are now unnecessary.

OTOH They may be useful if we ever decide to do anything with the currently unused:
* `(int) Host::mRetries`
* `(int) Host::mTimeout`